### PR TITLE
Fix #574: Vite update docs for MPA

### LINKED ISSUE #574: Vite 5: Returns 200 OK instead of 404 Not Found
Vite 5 now fails because by default `appType: 'spa'` does not return 404's if something isn't found and returns 200 and sends to index HTML bypassing the SPARouting.

https://vitejs.dev/config/shared-options#apptype

To fix it we need to tell vite this is a MultiPageApplication using `appType: 'mpa'` to make it work like Vite 4 with Quinoa.

```js
import react from "@vitejs/plugin-react";
import { defineConfig } from "vite";
import eslint from "vite-plugin-eslint";

// https://vitejs.dev/config/
export default defineConfig({
	plugins: [react(), eslint()],
	appType: "mpa",
});
```

### PR DESCRIPTION
**Pull request by melloware**
Fix #574: Vite update docs for MPA

### CODE REVIEW

File: docs/modules/ROOT/pages/web-frameworks.adoc [hunk 1]
```diff
@@ -206,6 +206,24 @@ This will configure the build to export the static files:
   }
 ----
 
+[#vite-config]
+=== Vite Configuration
+
+You must update `vite.config.ts` to let Vite know this is a Multiple Page Application so any unknown URL gets a `404` instead of routing to `index.html`.
+.vite.config.ts:
```
Comment (shivam-sharma7) [inline docs/modules/ROOT/pages/web-frameworks.adoc:213]:
```suggestion
vite.config.ts:
```
remove .

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/web-frameworks.adoc (hunk 1 above)
Comment (ia3andy) [reply docs/modules/ROOT/pages/web-frameworks.adoc:213]: The dot is important for asciidoc code block title

**Comment by shivam-sharma7** [comment]
@melloware why?

**Comment by melloware** [comment]
Superseded by https://github.com/quarkiverse/quarkus-quinoa/pull/593

**Pull request closed without merging**